src/ontology/mondo-edit.obo

```diff
@@ -293933,20 +293913,36 @@ property_value: confidence "0.7021739130434781" xsd:double
 
 [Term]
 id: MONDO:0014700
-name: neurodevelopmental disorder-craniofacial dysmorphism-cardiac defect-hip dysplasia syndrome due to a point mutation
+name: Au-Klein syndrome
 subset: ordo_etiological_subtype {source="Orphanet:453504"}
-synonym: "AU-Kline syndrome" RELATED [MONDO:Lexical, OMIM:616580]
-synonym: "Au-Kline syndrome" EXACT [Orphanet:453504]
-synonym: "AUKS" RELATED ABBREVIATION [MONDO:Lexical, OMIM:616580]
-synonym: "neurodevelopmental disorder-craniofacial dysmorphism-cardiac defect-hip dysplasia syndrome due to a point mutation" EXACT CLINGEN_PREFERRED []
+subset: ordo_malformation_syndrome {source="Orphanet:2729"}
+synonym: "Au-Kline syndrome" EXACT [OMIM:616580, Orphanet:453504]
+synonym: "AUKS" EXACT ABBREVIATION [MONDO:Lexical, OMIM:616580]
+synonym: "congenital hydronephrosis with cleft palate, characteristic facies, hypotonia, and intellectual disability" RELATED [GARD:0004064]
+synonym: "congenital hydronephrosis with cleft palate, characteristic facies, hypotonia, and mental retardation" RELATED DEPRECATED [GARD:0004064]
+synonym: "hydronephrosis, congenital, with cleft palate, characteristic facies, hypotonia, and intellectual disability" EXACT [OMIM:604916]
+synonym: "hydronephrosis, congenital, with cleft palate, characteristic facies, hypotonia, and mental retardation" EXACT DEPRECATED [OMIM:604916]
+synonym: "neurodevelopmental disorder-craniofacial dysmorphism-cardiac defect-hip dysplasia syndrome due to a point mutation" EXACT CLINGEN_PREFERRED [Orphanet:453504]
+synonym: "Okamoto syndrome" EXACT [OMIM:604916]
+xref: ICD10CM:Q87.8 {source="Orphanet:2729/attributed", source="Orphanet:2729/ntbt", source="Orphanet:2729"}
+xref: MESH:C565736 {source="MONDO:equivalentTo"}
+xref: OMIM:604916 {source="MONDO:equivalentObsolete", source="Orphanet:2729/e", source="Orphanet:2729"}
 xref: OMIM:616580 {source="MONDO:equivalentTo", source="Orphanet:453504", source="Orphanet:453504/e"}
+xref: Orphanet:2729 {source="OMIM:604916"}
```

Comment:
How does this line relate to Line 293929 since the Orphanet and OMIM identifiers are the same but only the `xref: OMIM:604916` has `source="MONDO:equivalentObsolete`?

Reply:
this OMIM class was merged into [OMIM:616580](https://omim.org/entry/616580), hence we merged it in Mondo. But yes, Orphanet does have this class as equiv to the OMIM class, so Orphanet should probably merge these terms too. They only do releases once a year, I believe, so we probably won't see this merge in Orphanet until their release.